### PR TITLE
Adds save property to data + access from save-view

### LINKED ISSUE #56: Make Saved view

### PR DESCRIPTION
**Pull request by Bokaiii**
closes #56

### CODE REVIEW

File: components/views/save.js [hunk 1]
```diff
@@ -1,13 +1,61 @@
...
-  );
-}
+  </View>
+);
+
+export default class SavedScreen extends React.Component {
+  constructor() {
+    super();
+    this.state = {
+      SaveData: [],
```
Comment (Joergenson) [inline components/views/save.js:25]:
```suggestion
      savedData: [],
```
seems you made little typo

File: components/views/save.js [hunk 2]
```diff
@@ -1,13 +1,61 @@
...
+  }
+
+  componentDidMount() {
+    this.getData();
+  }
+
+  render() {
+    return (
+        <FlatList
+          data={this.state.SaveData}
```
Comment (Joergenson) [inline components/views/save.js:52]:
```suggestion
          data={this.state.savedData}
```

**Review by Joergenson — CHANGES_REQUESTED** [review]
:-)

File: package.json [hunk 3]
```diff
@@ -21,6 +21,7 @@
     "@react-navigation/native": "^6.0.6",
     "expo": "~42.0.1",
     "expo-app-loading": "1.1.2",
+    "expo-cli": "^4.12.12",
```
Comment (kepet19) [inline package.json:24]: Whoops accidentally add expo-cli

**Review by kepet19 — CHANGES_REQUESTED** [review]

File: components/views/save.js [hunk 4]
```diff
@@ -1,13 +1,100 @@
...
+    </View>
+  </View>
+);
+
+function remove(id) {
+  console.log("Remove " + id);
 }
 
-export default SaveScreen;
+export default class SavedScreen extends React.Component {
```
Comment (Joergenson) [inline components/views/save.js:62]:
```suggestion
export default function SavedScreen() {
  const [saveData, setSaveData] = useState([]);
  const [fetching, setFetching] = useState(false);

  const getData = () => {
    Database.getItem("data").then((data) => {
      let filtered = data.filter((items) => items.saved);
      setSaveData(filtered);
    });
  }

  const onRefresh = () => {
    setFetching(true);
    getData();
    setFetching(false);
  }

  useEffect(() => {
    getData();
  }, []);

  return (
    <FlatList
      data={saveData}
      extraData={saveData}
      keyExtractor={(item) => item.id.toString()}
      renderItem={renderSaveScreen}
      refreshing={fetching}
      onRefresh={() => onRefresh}
    />
  );
}

```
you can adapt it like this :)

**Review by kepet19 — APPROVED** [review]

**Comment by Kechr** [comment]
@Joergenson Have your requested changes been resolved?

**Comment by Joergenson** [comment]
> @Joergenson Have your requested changes been resolved?

ye

**Review by Joergenson — APPROVED** [review]

**Pull request merged**